frontend/scripts/react-components/tool/sankey/sankey.selectors.js

```diff
@@ -200,6 +212,12 @@ export const getSankeyLinks = createSelector(
       return null;
     }
 
+    // Fix a race condition in which the links were still loading
```

Comment:
i think this race condition will be gone if we manage to detach the links loading from the columns loading.